contributed_definitions/nyaml/NXsample_substrate.yaml

```diff
@@ -0,0 +1,103 @@
+category: base
+doc: |
+  Any information on the sample substrate.
+type: group
+NXsample_substrate(NXobject):
+  material(NX_CHAR):
+    doc: |
+      Material of the substrate in direct contact with the sample.
+  state(NX_CHAR):
+    doc: |
+      Physical state of the substrate, similar options to sample state
+  notes(NXnote):
+    doc: |
+      Further notes.
+  (NXsubstance):
+    doc: |
+     Any substance contained in the sample substrate.  
+  (NX_fabrication):
+    doc: |
+      Name of the sample substrate vendor (company or research group)
+  depends_on(NX_CHAR):
+    doc: |
+      NeXus positions components by applying a set of translations and rotations
+      to apply to the component starting from 0, 0, 0. The order of these operations
+      is critical and forms what NeXus calls a dependency chain. The depends_on
+      field defines the path to the top most operation of the dependency chain or the
+      string "." if located in the origin. Usually these operations are stored in a
+      NXtransformations group. But NeXus allows them to be stored anywhere.
+  (NXtransformations):
+    doc: |
+      This is the group recommended for holding the chain of translation
+      and rotation operations necessary to position the component within
+      the instrument. The dependency chain may however traverse similar groups in
+      other component groups
```

Comment:
What is the underlying idea of sample composition? What is connected to what? I'm not sure I understand this entirely.

Reply:
This has been removed and should just be another component, based on your previous comment.